### DISCUSSION TITLE
Want to contribute

### DISCUSSION THREAD

**Discussion by KhageshorGiri** [dotnet-architecture/eShopOnContainers#2006] [Q&A] [answered]
#### Hello all, 
My name is **Khageshor Giri**. 
I am a *software engineer* form **Nepal** but I am **new open source**. 
I want to contribute in this project. 
Please provide me some guidelines. 

**Thank You.**

**Comment by erjain** [comment, marked as answer]
Hi @KhageshorGiri, thank you so much for showing interest in this project. 
You can always contribute here, we accept pull requests from the community. When doing it, you can fork your own copy, please do it in the DEV branch which is the consolidated work-in-progress branch, contains the latest code, and is the branch actively developed. 
Note that all PRs must be against the dev branch to be considered. This branch is developed using .NET 6.

You can begin with [README.md](https://github.com/dotnet-architecture/eShopOnContainers#readme) and for more information, you can follow our wiki page: https://github.com/dotnet-architecture/eShopOnContainers/wiki. I hope it helps you to start, please feel free to share any queries.
